### PR TITLE
Fix tailwind feature googleapi setup

### PR DESCRIPTION
**Pull request by davirst**
## Description of what was implemented
Fixing tailwind feature google API setup
-

## Testing Notes:
Notes for any specific setup to get the branch working

------

## Checklist
- [x] Description section in merge request filled
#### Branching
- [x] Source branch was branched off of develop
- [x] Target branch is merging back to develop
- [x] Branch is correctly named
    - prefixed with `feature-`, `fix-`, `refactor-`
#### Code Quality
- [x] Lint Passes (yarn lint)
    - `npm run lint`
- [x] All comments are clean, concise and relevant
  * Comments are only for code blocks that need it
  * No code blocks are commented out
  * No imports are commented out
  * No debugging comments

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -20,20 +20,31 @@ module.exports = {
       'secondBlack': '#00204A',
     },
 
+    // width: {
```
Comment (Dada-Tech) [inline tailwind.config.js]: no comments in code please

File: tailwind.config.js [hunk 2]
```diff
@@ -20,20 +20,31 @@ module.exports = {
...
+      lg: '1024px',
+      xl: '1440px',
+
       padding: {
         DEFAULT: '1rem',
         sm: '2rem',
         lg: '4rem',
         xl: '5rem',
         '2xl': '6rem',
+        'px-4': '22rem, 19rem',
```
Comment (Dada-Tech) [inline tailwind.config.js]: why does this need to be there ? wont it override px-4 for all styles?

**Review by Dada-Tech — COMMENTED** [review]
a couple comments.

i've also deleted the env file then committed the change. do the same in the other branch so its not in the merge request please

File: tailwind.config.js (hunk 2 above)
Comment (davirst) [reply tailwind.config.js]: It's a mistake I did it once and cleaned it but came back. Will take it out immediately

**Pull request merged**